lifecycle/src/main/resources/action/feed/eviction-action.xml

```diff
@@ -39,6 +39,7 @@
             </property>
```

Comment:
If you are setting the launcher memory for others, better be consistent and set it here too.